helpers/scss.js

```diff
@@ -1,13 +1,14 @@
-module.exports = function(gulp, plugins, config, name, locale, file) {
+'use strict';
+module.exports = function (gulp, plugins, config, name, locale, file) {
   // Return function that is executed inside of .pipe()
   return () => {
-    var theme      = config.themes[name],
-        src        = file || config.projectPath + theme.src + '/**/*.scss',
-        dest       = config.projectPath + theme.dest + '/' + locale,
-        maps       = plugins.util.env.maps || false,
-        production = plugins.util.env.prod || false,
-        postcss    = [],
-        parentPath = require('./parent-theme-dir')(name, config);
+    const theme      = config.themes[name],
+          src        = file || config.projectPath + theme.src + '/**/*.scss',
+          dest       = config.projectPath + theme.dest + '/' + locale,
+          maps       = plugins.util.env.maps || false,
+          production = plugins.util.env.prod || false,
+          postcss    = [],
```

Comment:
Thanks for clarifying, I'm new in ES6 and thought const is const, not almost const 😄